### PR TITLE
changes to Did You Know section

### PR DESCRIPTION
**Pull request by jacksdeguzman**
Added text and video to 'Did You Know' section and amended css.  I can't for the life of me get the list and the video to appear in one row! grrr...

### CODE REVIEW

**Comment by arevives** [comment]
LGTM !!!

**Review by arevives — COMMENTED** [review]
lgtm!

**Review by arevives — COMMENTED** [review]
lgtm!

**Review by arevives — COMMENTED** [review]
lgtm wow!

File: index.html [hunk 1]
```diff
@@ -110,8 +110,19 @@ <h6 class="hero-text-sub">
...
+            <h2>Did You Know</h2>
+            <div class="flex-container">
+                <div class="counter-list-1">
+                  <ul>
+                    <li><span class="counter">39%</span> of students in college experience a significant mental health issue.</li>
+                    <li><span class="counter">1 in 5</span> young adults experiences a mental health condition</li>
+                    <li><span class="counter">15%</span> of college students suffer from anxiety</li>
+                  </ul>
+                </div>
+                <div><iframe width="480" height="260" src="https://www.youtube.com/embed/0oxNRaPKcHE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (sbuckingham89) [inline index.html:123]: @jacksdeguzman I think this is the one missing the closing tag

**Review by sbuckingham89 — APPROVED** [review]
Just need to add that closing div! and then this will be good!

**Pull request merged**